lib/veeqo/request.rb

```diff
@@ -63,6 +64,11 @@ def put(path, params = {})
         build_response_object response
       end
 
+      def quantity(path, params = {})
+        response = raw_request(:get, path, params)
+        response.headers['X-Total-Count'].to_i
```

Comment:
👍

Reply:
@yurkiv I think 0 as a total count makes more sense then just not found returned.
For example, I would like to know, how many requests are proceed to Veeqo to take all products with 100 items per page pagination.
I take Veeqo::Product.count / 100 and get that number.
If it produces 404 error, than I have to rescue that error. That's extra code here, but if it breaks gem's logic, I can raise error if X-Total-Count is 0.